content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst

```diff
@@ -1,262 +1,321 @@
+date from the warehouse is on September 20th. Odoo uses lead times and automated reordering rules to
+schedule the necessary operations based on the outgoing shipment delivery date, September 20th:
 
-The *Security Lead Time* for manufacturing allows generating
-manufacturing orders earlier to cope with the risk of manufacturing
-delays.
+.. image:: scheduled_dates/global-example.png
+   :align: center
+   :alt: Show timeline of how lead times work together to schedule warehouse operations.
 
-To enable it, go to :menuselection:`Manufacturing --> Configuration -->
-Settings` and tick *Security Lead Time*.
+-  **September 1st**: Sales order created, confirmed by salesperson
 
-.. image:: scheduled_dates/scheduled_dates_10.png
-    :align: center
-    :alt: View of the security lead time for manufacturing from the manufacturing app settings
+-  **September 9th**: Deadline to order components to ensure they arrive in time when manufacturing
+   begins (4-day supplier lead time)
 
-For example, a customer orders B with a delivery date scheduled on the
-20th of June. The *Manufacturing Lead Time* is 14 days and the
-*Security Lead Time* is 3 days, so the manufacturing of B needs to
-start at the latest on the 3rd of June, which is the MO’s planned date.
+-  **September 13th**: Scheduled date of receipt for components. Was initially set to 9/14, but the
+   1-day purchase security lead time pushed the date earlier by 1 day
 
-Global Example
-==============
+-  **September 14th**: Deadline to begin manufacturing. Calculated by subtracting  the manufacturing
+   lead time of 3 days and the manufacturing security lead time of 2 days from the expected delivery
+   date of 9/19
 
-Here is a configuration:
-
--  1 day of security lead time for Sales
--  2 days of security lead time for Manufacturing
--  3 days of manufacturing lead time
--  1 day of security lead time for Purchase
--  4 days of supplier lead time
-
-Let’s say that a customer orders B on the 1st of September and the
-delivery date is planned to be within 20 days (September 20th). In such
-a scenario, here is when all the various steps are triggered.
-
--  **September 1st**: the sales order is created
--  **September 10th**: the deadline to order components from the supplier
-   because of the manufacturing process (4 days of supplier lead
-   time)
--  **September 13th**: the reception of the product from the supplier (1 day
-   of security lead time for Purchase)
--  **September 14th**: the deadline start date for the manufacturing (19th -
-   3 days of manufacturing lead time - 2 days of security lead time
-   for Manufacturing)
--  **September 19th**: the expected date on the delivery order form (1 day
-   of security lead time for sales)
+-  **September 19th**: :guilabel:`Scheduled Date` on the delivery order form indicates the updated
+   expected delivery date, which was originally set as 9/20. But the sales security lead time pushed
+   the date forward by a day.
```

Comment:
I think a nice closing sentence or paragraph after this would help to tie up the example or doc as a whole. What do you think?

Reply:
Will think about what to write! Thanks for such a thorough review, Brandon (: